JavaScript/Topics/17.jQuery/README.md

```diff
@@ -0,0 +1,131 @@
+# What is jQuery ?
+
+  Query is a fast, small, cross-platform and feature-rich JavaScript library. It is designed to simplify the client-side scripting of HTML.jQuery is a lightweight, "write     less, do more", JavaScript library.jQuery takes a lot of common tasks that require many lines of JavaScript code to accomplish, and wraps them into methods that you can call with a single line of code.jQuery also simplifies a lot of the complicated things from JavaScript, like AJAX calls and DOM manipulation.
+
+  The purpose of jQuery is to make it much easier to use JavaScript on your website.
+
+  The jQuery library contains the following features:<br>
+  <ul>
+  <li> HTML/DOM manipulation</li>
+  <li> CSS manipulation</li>
+  <li> HTML event methods</li>
+  <li> Effects and animations</li>
+  <li> AJAX</li>
+  <li> Utilities</li>
+  </ul>
+
+# Is jQuery a programming language?
+  jQuery is not a programming language but a well-written JavaScript code. It is used to traverse documents, event handling, Ajax interaction, and Animation.
+
+# What is the difference between JavaScript and jQuery?
+  The simple difference is that JavaScript is a language while jQuery is a built-in library built for JavaScript. jQuery simplifies the use of JavaScript language.  
+
+# Why jQuery is required ?
+  <ul>
+    <li> It is very fast and extensible.</li>
+    <li>  It facilitates the users to write UI related function codes in minimum possible lines.</li>
+    <li>  It improves the performance of an application.</li>
+    <li>  Browser's compatible web applications can be developed.</li>
+    <li>  It uses mostly new features of new browsers.</li>
+    <li>  So, you can say that out of the lot of JavaScript frameworks, jQuery is the most popular and the most extendable.
+            Many of the biggest companies on the web use jQuery.</li>
+  </ul>
+
+  Some of these companies are:<br>
+  <ul>
+    <li> Microsoft</li>
+    <li> Google</li>
+    <li> IBM</li>
+    <li> Netflix</li>
+  </ul>
+
+# What should you know before starting to learn jQuery?
```

Comment:
Change all of `#` to `##`

Reply:
@dakshsethi Sir I add ## in documentation . Any other changes